### PR TITLE
Issue hla 255 - Address issues associated with routines under copyright

### PR DESCRIPTION
**Pull request by mdlpstsci**
Modified HSTCAL infrastructure routines for ACS, WFC3, and STIS which have copyright issues.

### CODE REVIEW

File: pkg/acs/calacs/History [hunk 1]
```diff
@@ -1,3 +1,6 @@
+###  01-Sep-2019 - MDD -- Version 10.2.1
+    Removed/modified/updated routines which used code from numerical recipes.
```
Comment (stsci-hack) [inline pkg/acs/calacs/History]: Would this be better stated "used copyrighted code"?

File: pkg/wfc3/calwf3/Dates [hunk 2]
```diff
@@ -1,3 +1,9 @@
+01-Sep-2019 - MDD - Version 3.5.1
+    - Replaced/modified routines based upon copyright numerical recipes routines.
```
Comment (stsci-hack) [inline pkg/wfc3/calwf3/Dates]: Would this be better stated as simply "based upon copyrighted routines"?

File: pkg/wfc3/calwf3/Updates [hunk 3]
```diff
@@ -1,3 +1,9 @@
+Updates for Version 3.5.1 01-Sep-2019 - MDD
+    - Replaced/modified routines based upon copyright numerical recipes routines.
```
Comment (stsci-hack) [inline pkg/wfc3/calwf3/Updates]: Another mention of numerical recipes...

**Comment by mdlpstsci** [comment]
@hack I made the updates to the documentation.  I do not have permission to merge.

**Comment by pllim** [comment]
Tests are failing. Are they related? :eyes:

**Comment by mdlpstsci** [comment]
> Tests are failing. Are they related? 👀

No.  I have to ask Joe.

**Comment by pllim** [comment]
If you rebase now, you would pick up the fix from #410 that has been merged. Then, if everything goes well, Jenkins should pass. Ignore Travis CI for now (#411).

File: pkg/acs/calacs/History (hunk 1 above)
Comment (mdlpstsci) [reply pkg/acs/calacs/History]: Done

File: pkg/wfc3/calwf3/History [hunk 4]
```diff
@@ -1,3 +1,9 @@
+### 01-Sep-2019 - MDD - Version 3.5.1
+    - Replaced/modified routines based upon copyright numerical recipes routines.
```
[the remark this answers is not in the document]
Comment (mdlpstsci) [reply pkg/wfc3/calwf3/History]: Done.

File: pkg/wfc3/calwf3/Dates (hunk 2 above)
Comment (mdlpstsci) [reply pkg/wfc3/calwf3/Dates]: Done

File: pkg/wfc3/calwf3/Updates (hunk 3 above)
Comment (mdlpstsci) [reply pkg/wfc3/calwf3/Updates]: Done

**Pull request merged**